core/cloudflow-localrunner/src/main/scala/cloudflow/localrunner/LocalRunner.scala

```diff
@@ -131,7 +131,7 @@ object LocalRunner extends StreamletLoader {
                             StreamletDeployment.EndpointContainerPort + endpointIdx)
       deployment.endpoint.foreach(_ => endpointIdx += 1)
 
-      val runnerConfigObj = RunnerConfig(appId, appVersion, deployment, Some(bootstrapServers))
+      val runnerConfigObj = RunnerConfig(appId, appVersion, deployment)
```

Comment:
I think this needs a little more work for runLocal.

Reply:
:+1: